orca-queue/src/main/kotlin/com/netflix/spinnaker/config/QueueConfiguration.kt

```diff
@@ -61,6 +61,8 @@ open class QueueConfiguration {
       ThreadPoolTaskExecutor().apply {
         corePoolSize = 20
         maxPoolSize = 150
+        setQueueCapacity(20)
```

Comment:
should queue capacity be 0? I think we don't want to queue anything. or if this sizes the BlockingQueue then maybe set it to 1 instead of 20?